hbase_sql/src/main/java/com/splicemachine/hbase/SpliceMasterObserver.java

```diff
@@ -270,7 +266,7 @@ public void shutdown() throws Exception {
             try {
                 MasterLifecycle distributedStartupSequence = new MasterLifecycle();
                 manager.registerEngineService(new EngineLifecycleService(distributedStartupSequence, config, true));
-                manager.start();
+                manager.start(null);
```

Comment:
@jyuanca Why we don't need to set replication role for MasterObserver?

Reply:
This flag is to prevent a writeable transaction. No one can connect to master and run write workload